src/templates/_common/components/forms.jsx

```diff
@@ -227,10 +240,20 @@ export const SubmitButton = ({
         </React.Fragment>;
 
     if (!no_wrapper) {
+
+        const getClassName = () => {
+            if (!is_left_align) {
+                return 'gr-8 gr-push-4 gr-12-m gr-push-0-m';
+            } else if (left_alignment_size) {
+                return `gr-${12 - left_alignment_size} gr-push-${left_alignment_size} gr-12-m gr-push-0-m`;
+            }
+            return '';
+        };
+
         return (
             <div className={`${is_centered ? 'center-text' : 'gr-row'} ${className || ''}`} id={id}>
                 { !is_centered ?
-                    <div className={`${!is_left_align ? 'gr-8 gr-push-4 gr-12-m gr-push-0-m' : ''} center-text-m`}>
+                    <div className={`${getClassName()} center-text-m`}>
```

Comment:
I wanted to ask, what does this function exactly do? When I look in the file, I see this function nowhere imported/passed or created. How does it work?